### PR TITLE
solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**
https://yelnikov-andrii.github.io/layout_miami/

### CODE REVIEW

**Comment by Denchepornyuk** [comment]
Screenshot 2022-04-15 at 11 00 03

something is wrong with the image, fix the image size

**Comment by Denchepornyuk** [comment]
Screenshot 2022-04-15 at 11 05 49

fix the text in browser tab

**Comment by Denchepornyuk** [comment]
Screenshot 2022-04-15 at 11 06 26

fix images, check images in the design

**Comment by Denchepornyuk** [comment]
Screenshot 2022-04-15 at 11 07 38

fix images width on small screen, check images in the design

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,281 @@
...
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a class="logo">
+              <img src="./images/logo.svg"
+                   alt="bose"
+                   class="logo__img"
+              >
```
Comment (Denchepornyuk) [inline src/index.html:20]: fix intends in all places
```suggestion
              <img
                src="./images/logo.svg"
                alt="bose"
                class="logo__img"
              >
```

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]
good job

**Comment by Denchepornyuk** [comment]
Screenshot 2022-04-15 at 17 22 40

add a small hover effect for images
a small scale would be great

**Comment by Denchepornyuk** [comment]
Screenshot 2022-04-15 at 17 25 46

дуже рано робиш мобільну версію для сторінки, зробити сайт більшим на ширині між 480 і 768 px, а від 480 вже залишай мобільну версію

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! 👍 
Some comments:
1) Don't forget to add favicon to your page
2) Logo in header should be the link (you can add # in href for it)
3) Will be better to set height of header = 100vh, now I don't see full title text
image
4) Add some styles on hover state for images in 'recommended' section
5) Add some styles on hover state for links near contact form
6) Address link near the contact form should be opened in the new tab

**Pull request closed without merging**